plugins/carbon_black_response/icon_carbon_black_response/actions/uninstall_sensor/action.py

```diff
@@ -0,0 +1,39 @@
+import cbapi.errors
+import insightconnect_plugin_runtime
+from .schema import UninstallSensorInput, UninstallSensorOutput, Input, Output, Component
+from insightconnect_plugin_runtime.exceptions import PluginException, ConnectionTestException
+from cbapi.errors import ObjectNotFoundError, ApiError, UnauthorizedError, ServerError
+
+
+# Custom imports below
```

Comment:
please delete this comment. Since we use `black` this separation does not make sense.